Sources/Utils/Backtrace.cpp

```diff
@@ -67,13 +69,76 @@ void PrintBacktrace() {
   }
 }
 #elif defined(OS_WIN32)
+
+static HANDLE CurrentProcess() {
+  // Based on
+  // https://learn.microsoft.com/en-us/windows/win32/debug/initializing-the-symbol-handler
+  HANDLE process = GetCurrentProcess();
+  // Explanation for this is at
+  // https://github.com/apple/swift/blob/0c56f1468c0f56ab9b96464ba0d63f7db4b84b69/stdlib/public/runtime/ImageInspectionCOFF.cpp#L42-L53.
+  if (!DuplicateHandle(process, process, process, &process, 0, false,
+                       DUPLICATE_SAME_ACCESS)) {
+    DS2LOG(Error, "failed to dup current process handle; proceeding anyway");
+  }
+  if (SymInitialize(process, NULL, TRUE)) {
+    return process;
+  }
+  // SymInitialize failed
+  auto error = GetLastError();
```

Comment:
Can we inline this into the `DS2LOG` below?

Reply:
Done.